### ISSUE TITLE
Demo should autofill

### ISSUE THREAD

**Issue by tjsavage** [PolymerElements/gold-cc-input#31]
I think it doesn't b/c it's not in a form

**Comment by notwaldorf** [comment]
@tjsavage True. Should we change all the gold-* element demos to be actual forms? Or should we just link to https://solid-gold.firebaseapp.com/ in all of the demos?
